src/components/map/Layers/Countries.tsx

```diff
@@ -0,0 +1,113 @@
+import { useState, useEffect, useContext } from "react"
+import { AppContext } from "context";
+import ReactDOMServer from "react-dom/server";
+import { EstimateGradePrevalence } from "types";
+import mapboxgl from "mapbox-gl";
+import CountryPopup from 'components/map/Popups/CountryPopup'
+
+function SetCountryEstimates(map: mapboxgl.Map, estimateGradePrevalences: EstimateGradePrevalence[]) {
```

Comment:
Can you add a few quick comments explaining the distinction between SetCountryEstimates and UpdateCountryEstimates here?

Seems like UpdateCountryEstimates is actually just performing filtering and not actually updating the estimates associated with each country?